README.md

```diff
@@ -1,7 +1,35 @@
-# Settings View package [![Build Status](https://travis-ci.org/atom/settings-view.svg?branch=master)](https://travis-ci.org/atom/settings-view)
+# Settings View [![Build Status](https://travis-ci.org/atom/settings-view.svg?branch=master)](https://travis-ci.org/atom/settings-view)
 
-Edit config settings, install packages, and change themes in Atom.
+Edit config settings, install packages, and change themes all from within Atom.
 
-Open by pressing `cmd-,` or selecting the _Atom > Preferences..._ menu.
+## Usage
+You can open the Settings View by navigating to _Atom > Preferences_ (Mac), _File > Preferences_ (Windows), or _Edit > Preferences_ (Linux).
 
-![screen shot 2015-01-05 at 8 08 18 pm](https://cloud.githubusercontent.com/assets/1476/5624241/d29ffbe6-9516-11e4-8bfe-e91623977bc7.png)
+In order to install new packages and themes, simply click on the _Install_ section located on the left-hand side.
+Once installed, community packages/themes (and their settings) are housed within their respective section.
+All packages/themes that have updates will be listed under the _Updates_ section.  Finally, all keybindings (including ones that community packages have added) are available in the _Keybindings_ section.
+
+Want to learn more?  You may want to check the [Getting Started: Atom Basics](https://atom.io/docs/latest/getting-started-atom-basics#settings-and-preferences) and [Using Atom: Atom Packages](https://atom.io/docs/latest/using-atom-atom-packages) sections in the flight manual.
+
+### Commands
+* `settings-view:open` - Opens up the Settings View
+* `settings-view:show-keybindings` - Opens up the _Keybindings_ section within the Settings View
+* `settings-view:uninstall-packages` - Opens up the _Packages_ section within the Settings View
+* `settings-view:change-themes` - Opens up the _Themes_ section within the Settings View
+* `settings-view:uninstall-themes` - Opens up the _Themes_ section within the Settings View
+* `settings-view:check-for-updates` - Opens up the _Updates_ section within the Settings View
+* `settings-view:install-packages-and-themes` - Opens up the _Install_ section within the Settings View
```

Comment:
What do you think about using "Shows" instead of "Opens up", and "of Settings View" instead of "within the Settings View"? :thought_balloon: